### PR TITLE
add sharma ICPC 2022 papers

### PR DESCRIPTION
**Pull request by rishab-32**
Adds two ICPC 2022 papers.

### CODE REVIEW

File: _publications/sharma2022an.markdown
Comment (mallamanis) [inline _publications/sharma2022an.markdown:1]: Can you rename this to `sharma2022exploratory.markdown`?

**Review by mallamanis — CHANGES_REQUESTED** [review]
Thanks a lot! One minor change and we can merge this.

File: _publications/sharma2022an.markdown
Comment (rishab-32) [reply _publications/sharma2022an.markdown:1]: Updated. Thanks.

**Comment by mallamanis** [comment]
Thanks!

**Pull request merged**